test/test_nestedtensor.py

```diff
@@ -4312,6 +4323,132 @@ def test_softmax_reduce_batch_dim(
             ):
                 out = torch.nn.functional.softmax(nt, dim=reduce_dim)
 
+    @dtypes(torch.float32)
+    @parametrize("requires_grad", [False, True])
+    @parametrize("components_require_grad", [False, True])
+    def test_layer_norm_reduce_ragged_idx_1(
+        self, device, dtype, requires_grad, components_require_grad
+    ):
+        """
+        Layer normalization on NestedTensor passes when trying to normalize across ragged dimension, where ragged_idx == 1.
+        """
+        if (
+            torch._dynamo.is_compiling() and not requires_grad
+        ):  # requires_grad = False does not work with dynamo
```

Comment:
note for myself: come back to check this

Reply:
okay, this is probably fine but can you add a quick comment explaining why you need this?

what I get is
```
AssertionError: SymInts must use SymNodeVariable. If the underlying value is static, we will create a ConstantVariable and specialize.
```
from
```
   File "/data/users/dberard/pytorch/test/test_nestedtensor.py", line 4362, in torch_dynamo_resume_in_test_layer_norm_reduce_ragged_idx_1_at_4342
    *nt.shape[nt._ragged_idx + 1 :],
```

Reply:
yes definitely!